src/Pages/HeungsooLee/Login/Login.js

```diff
@@ -1,21 +1,182 @@
-import React from 'react'
-import './Login.scss'
-import '../../../styles/reset.scss'
-import '../../../styles/common.scss'
-import { withRouter } from 'react-router-dom';
+import React from "react";
+import { withRouter } from "react-router-dom";
+import "./Login.scss";
+import "../../../styles/reset.scss";
+import "../../../styles/common.scss";
 
+const footerItem = [
```

Comment:
이렇게 변하지 않는 상수는 대문자_대문자로 적어주세요!
```suggestion
const FOOTER_ITEM = [
```